### ISSUE TITLE
Question about the API document of convolution

### ISSUE THREAD

**Issue by JacobIsrael123** [tensorflow/tensorflow#667]
In [issue 196](https://github.com/tensorflow/tensorflow/issues/196), the problem of definition of different padding is mentioned. 
https://www.tensorflow.org/versions/master/api_docs/python/nn.html#convolution

I think the document of convolution is still quite confusing right now. 
What does the different dimension of ```strides = [1, 1, 1, 1]``` mean?
Why the definition of "VALID" and "SAME" padding is quite different from the [Stanford cs131n](http://cs231n.github.io/convolutional-networks/) tutorial. 

I think most of the people learn CNN according to the tutorials online. Is it possible to modify the document according to the tutorial?  Thank you very much.

**Comment by cesarsalgado** [comment]
The strides = [1, 2, 2, 1] are the strides for each of the dimension of the input tensor [batch, in_height, in_width, in_channels]. For the tf.nn.conv2d function the dimension 0 and 3 must have strides equal to 1. Based on what @Yangqing said in #196 I think the padding scheme is non-intuitive due to legacy reasons.

**Comment by girving** [comment]
Closing due to lack of activity.
